src/components/ui/date-range-picker/date-range-picker.tsx

```diff
@@ -46,6 +57,11 @@ export const DateRangePicker: React.FC<DateRangePickerProps> = ({
         maxDate={maxDate}
         containerClassName={containerClassName}
       />
+      {isErrorPresent && (
+        <p className='text-red-500 text-sm'>
+          {`${error?.start_date} date and ${error?.end_date} date schedule is required`}
```

Comment:
Please remove the word `schedule` here

Reply:
Also, there are different errors present, not just date is required. Please handle these cases as well